src/main/java/at/gammastrahlung/monopoly_server/network/websocket/WebSocketSender.java

```diff
@@ -16,6 +16,7 @@
 
 @NoArgsConstructor
 public class WebSocketSender {
+    private static Player playersforAuction;
```

Comment:
Remove this, websocketsender should only send messages to the specified players and should not handle anything else